civihr_employee_portal/civihr_leave_absences/templates/my_leave.html

```diff
@@ -24,26 +24,573 @@
+                              <td colspan="2" class="chr-grid__row__cell--light">Awaiting Approval</td>
+                              <td class="chr-grid__row__cell--light">-4</td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td><i class="fa fa-ellipsis-v" aria-hidden="true"></i></td>
+                            </tr>
+                            <tr class="chr-grid__row">
+                              <td colspan="2">Accrued TOIL(Expires 20/03/2015)</td>
+                              <td colspan="2" class="chr-grid__row__cell--light">18/01/2015</td>
+                              <td colspan="2" class="chr-grid__row__cell--light">More information required</td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td class="chr-grid__row__cell--light">1</td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td class="chr-grid__row__cell--light"></td>
+                              <td><i class="fa fa-ellipsis-v" aria-hidden="true"></i></td>
+                            </tr>
+                          </tbody>
+                        </table>
+                      </div><!--end of .table-responsive-->
+                    </div>
+                  </div>
+                </div><!--panel body -->
+              </div>
+            </div>
+            <div class="row chr-container__row chr-container__row--summary ">
+            <div class="table-responsive">
+              <table class="table table-borderless">
+                <thead>
+                  <tr>
+                    <th colspan="6">
+                    <p >Future Balance</p>
+                    </th>
+                    <th>15</th>
+                    <th>2</th>
+                    <th></th>
+                    <th></th>
+                    <th></th>
+                    <th></th>
+                  </tr>
+                </thead>
+              </table>
+            </div>
+            </div>
+            <div class="row panel panel-default chr-grid">
+              <div class="panel-heading chr-grid__panel-heading">
+                <h4 class="panel-title">
+                    <div class="row ">
+                      <div class="table-responsive">
+                        <table class="table table-borderless">
+                          <thead>
+                            <tr class="civi-grid--open ">
+                              <th colspan="2">
+                                <i class="fa fa-chevron-down" aria-hidden="true"></i>
```

Comment:
You still have these `[aria-hidden="true"]` attributes. I keep pointing them out asking why you have them, but you neither remove them or you reply saying why you use them.

Either you explain why or you need to remove them